### PR TITLE
Feanil/update testing

### PR DESCRIPTION
**Pull request by feanil**

### CODE REVIEW

File: organizations/migrations/0001_initial.py [hunk 1]
```diff
@@ -36,7 +36,7 @@ class Migration(migrations.Migration):
                 ('modified', model_utils.fields.AutoLastModifiedField(default=django.utils.timezone.now, verbose_name='modified', editable=False)),
                 ('course_id', models.CharField(max_length=255, db_index=True)),
                 ('active', models.BooleanField(default=True)),
-                ('organization', models.ForeignKey(to='organizations.Organization')),
+                ('organization', models.ForeignKey(to='organizations.Organization', on_delete=models.CASCADE)),
```
Comment (ormsbee) [inline organizations/migrations/0001_initial.py]: Why this change to an existing migration?

Comment (ormsbee) [reply organizations/migrations/0001_initial.py]: Oh, nvm, didn't read the commit comment.

Comment (feanil) [reply organizations/migrations/0001_initial.py]: Was following the instructions from django's docs: https://docs.djangoproject.com/en/1.11/ref/models/fields/#django.db.models.ForeignKey.on_delete

on_delete will be a required field in django 2 and they want you to update both the model and the migration to retain the current behavior.

**Comment by feanil** [comment]
This has been pretty typical, though some are much further behind.  I'm trying to also add them to [the upgrade automation](https://github.com/edx/jenkins-job-dsl/blob/master/testeng/jobs/upgradePythonRequirements.groovy) so that they can start getting tested regularly if they have requirements updates.
